data/defscript/popups.kvs

```diff
@@ -1418,12 +1418,7 @@ defpopup(ignore)
 
 	label(%:ignLabelText)
 
-	item($tr("Enable Ignore","defscript"),$icon("ignore")) (!%:inIgnoreList && %:regName)
-	{
-		reguser.setIgnoreEnabled %:regName 1;
-	}
-
-	popup($tr("Ignore As...","defscript"),$icon("ignore")) (!%:inIgnoreList)
+	popup($tr("Ignore As...","defscript"),$icon("ignore")) (!%:inIgnoreList && !%:regName)
```

Comment:
Need to remove ` && !%:regName` as this breaks half the functionality of `Ignore As...`.